finrl/env/env_stocktrading_cashpenalty.py

```diff
@@ -208,7 +236,7 @@ def get_reward(self):
             assets = self.account_information['total_assets'][-1]
             cash = self.account_information['cash'][-1]
             cash_penalty = max(0, (assets*self.cash_penalty_proportion-cash))
-            assets -= cash_penalty
+            assets -= cash_penalty if assets > self.initial_amount else (self.negative_asset_penalty+1)*cash_penalty
```

Comment:
Sorry, not sure I'm following the logic here;
Why do we need extra penalty we are negative? This seems like we're inserting a human logic fallacy into the mix.

Reply:
Also, maybe I'm wrong here, so in that case, it might be good for us to have an easy way to choose a reward function or toggle between reward functions?

Reply:
The idea is to give a bigger penalty when we are short of cash and our current total asset is less than the initial amount. Yes, actually we can just turn that penalty off by assigning the `negative_asset_penalty` with 0

Reply:
Hey, how about we create a parameter for choosing the type of risk a user's can take (e.g. risk-averse, risk-moderate, risk-taker). What do you think? So it will be based on this `negative_asset_penalty` parameter.

cc: @XiaoYangLiu-FinRL @BruceYanghy

Reply:
I think this is an anthropomorphic assumption that I would want to see proven out. 
From a purely economics frame, the weighting of negative and positive consequences should allow the model to balance risk vs. reward properly. Intuitively it might be the case that adding steeper penalty for negative returns yields positive returns more often. However, there are a lot of variables at play, so we'd need to prove that. The common fallacy name is that 'losses loom larger than gains' which is why humans often get out of positions that they should hold when losses come in, and why people who don't invest their own money are often the most effective stewards of wealth. 
https://www.behavioraleconomics.com/resources/mini-encyclopedia-of-be/loss-aversion/#:~:text=Loss%20aversion%20is%20an%20important,as%20the%20pleasure%20of%20gaining.

Along these lines, I had actually contemplated adding a reward term related to sharpe ratio at any given step, or to concentration of holdings. However, my backtesting results have been very encouraging (>20% annual returns, >0.9 sharpe avg, beat market by 5% annual) so I'm inclined to leave this alone. 

For now, I do not agree with changing the reward function in any way given the complexity, and the unknowns. 
I'd suggest monkey patching the environment prior to training to make the change you're desiring here.

Reply:
Okay! I'll remove this in the next commit

Reply:
Thank you! Let's plan to collaborate on ideas of how to make the reward function configurable in the future!